packages/reactapp/src/NumberForm.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
```

Comment:
I see you are using this app for testing? Because working with state in storybook is a real pain?

Reply:
I was getting an issue in Storybook in which it wasn't setting the field default value which seemed to be something to do with Storybook and not formik or my code so it was easiest to put an example in the test app.